components/camel-kafka/src/main/java/org/apache/camel/processor/resume/kafka/KafkaResumeStrategyConfiguration.java

```diff
@@ -37,29 +38,23 @@ public Properties getProducerProperties() {
     }
 
     void setProducerProperties(Properties producerProperties) {
-        assert producerProperties != null;
-
-        this.producerProperties = producerProperties;
+        this.producerProperties = ObjectHelper.notNull(producerProperties, "producerProperties");
     }
 
     public Properties getConsumerProperties() {
         return consumerProperties;
     }
 
     void setConsumerProperties(Properties consumerProperties) {
-        assert consumerProperties != null;
-
-        this.consumerProperties = consumerProperties;
```

Comment:
Same as above

Reply:
My same reasoning as very top :point_up_2:.

Reply:
Same as the other one.